src/combine.cpp

```diff
@@ -694,10 +694,8 @@ void do_symbol_check(chunk_t *prev, chunk_t *pc, chunk_t *next)
 
    // separate the uses of CT_ASSIGN sign '='
    // into CT_ASSIGN_DEFAULT_ARG, CT_ASSIGN_FUNC_PROTO
-   if (  chunk_is_token(prev, CT_WORD)
+   if (  pc != nullptr && pc->parent_type == CT_FUNC_PROTO
```

Comment:
BTW, if you switched these (`chunk_is_token(PC, CT_ASSIGN) && pc->parent_type == CT_FUNC_PROTO`) you could omit the null check.

Reply:
done in 461de72